c/src/test/java/org/apache/arrow/c/RoundtripTest.java

```diff
@@ -77,6 +79,7 @@
 import org.apache.arrow.vector.ValueVector;
 import org.apache.arrow.vector.VarBinaryVector;
 import org.apache.arrow.vector.VarCharVector;
+import org.apache.arrow.vector.VariableWidthVector;
```

Comment:
Unintentional import? This is failing the build

Reply:
Fixed. Other test failures seems to be unrelated to this PR.